integration/integration_test.go

```diff
@@ -7409,11 +7420,19 @@ func startSSHServer(t *testing.T, caPubKeys []ssh.PublicKey, hostKey ssh.Signer)
 				}
 				if req.Type == sshutils.AgentForwardRequest {
 					agentForwarded = true
+				} else if req.Type == sshutils.ExecRequest {
+					_, err = channel.SendRequest("exit-status", false, binary.BigEndian.AppendUint32(nil, 0))
```

Comment:
Why are we always sending exit-code 0? I don't mind hardcoding a value in a test itself, but this is a test utility function used in many places. Hardcoding a value here maybe really confusing for someone who will use this function the next time.